src/locales/en/messages.json

```diff
@@ -1497,5 +1497,21 @@
   },
   "personalOwnershipPolicyInEffect": {
     "message": "An organization policy is affecting your ownership options."
+  },
+  "allSends": {
+    "message": "All Sends"
+  },
+  "sendTypeFile": {
+    "message": "File"
+  },
+  "sendTypeText": {
+    "message": "Text"
+  },
+  "searchSends": {
+    "message": "Search Sends",
+    "description": "'Send' is a noun and the name of a feature called 'Bitwarden Send'. It should not be translated."
+  },
+  "sendInformation": {
+    "message": "Send Information"
```

Comment:
man this is messing with me, is this labeling a "View Send"? If not, maybe "Send: Information" or "Send, Information", not sure. It's probably fine, but no matter what you need to copy the `"description"` above to anywhere we use the term "Send"

Reply:
Yes, that's in there right now for "View Send". In the Vault that page is labeled with "Item Information", so this is derivative of that. Luckily that string is sort of ahead of this PR, as "Viewing" a send is really just a screen that says "Send Information" on it right now.

Reply:
I added the "do not translate" description, but left the text as is for now since it isn't part of a testable feature yet.